app/controllers/bands_controller.rb

```diff
@@ -39,6 +39,26 @@ def update
     end
   end
 
+  def destroy_confirmation
+    find_band(params[:id])
+  end
+
+  def cannot_destroy
+    # This is a placeholder method to render the cannot_destroy template
+  end
+
+  def destroy
+    find_band(params[:id])
+
+    if @band.exemptions.any?
```

Comment:
I'd expect a dedicated method for checking whether the band can be deleted, something like @band.can_be_destroyed? also I'd also add a callback to execute this validation before destroy is triggered. This is to avoid accidental deletions (e.g. by cascading foreign key or some service that doesn't use this controller).

But as we discussed that previously, it makes sense to keep it simple for now till all requirements are clear